install-swagger-ui.sh

```diff
@@ -0,0 +1,8 @@
+# Copyright 2015, EMC, Inc.
+
+cd ./static
+rm -rf swagger-ui
+curl -L -o swagger-ui.zip https://github.com/zyoung51/swagger-ui/archive/master.zip
```

Comment:
@benbp I suggested forking into RackHD, but bintray is another option.   Looks like Zack is copying what @rolandpoulter did with web-ui. Should this be applied across the board for things like this?